ocs_ci/helpers/helpers.py

```
@@ -7669,3 +7669,251 @@ def delete_cephfs_ec_pool(pool_name):
+        if subvolume_path in root:
+            client_id = c["id"]
+            client_addr = c["inst"].split(" ")[-1]
+            break
+
+    assert (
+        client_id is not None
+    ), f"No CephFS client found for subvolume {subvolume_path}"
+    logger.info(f"Found client id={client_id}, addr={client_addr}")
+
+    toolbox.exec_sh_cmd_on_pod(command=f"ceph osd blocklist add {client_addr}")
+    logger.info(f"Blocklisted client {client_addr}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)